src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/URLIndexWFSHttp.java

```diff
@@ -0,0 +1,133 @@
+package au.org.emii.geoserver.wms;
+
+import org.geotools.util.logging.Logging;
+import org.joda.time.DateTime;
+
+import java.io.*;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class URLIndexWFSHttp implements URLIndexInterface {
```

Comment:
I think this should be UrkIndexWfsHttp. If you're happy with capitals for the acronyms then HTTP should probably be caps, too.